ONSdigital/eq-author-app
# Redirect users if they dont use Chrome

### What is the context of this PR?
Author breaks if user opens application in Internet Explorer.

Added a script in `index.html` to redirect users to `unsupported-browser.html` if they don't use Chrome.
Also added a script in `unsupported-browser.html` to redirect users back to Author if they are using Chrome.

### How to review
1. Open Author in Internet Explorer 11 (vm can be found here https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/)
2. You should be redirected to `http://localhost:3000/unsupported-browser.html`.
3. Copy the whole url in Internet Explorer and open it in Google Chrome.
4. You should be redirected back to the author application.
